test_convert.py

```diff
@@ -1,11 +1,14 @@
 import convert as cvt
 
 def test_valid():								
- 	assert cvt.convert(1) == 'I';
- 	assert cvt.convert(3) == 'III';
- 	assert cvt.convert(10) == 'X';
- 	assert cvt.convert(62) == 'LXII';
- 	assert cvt.convert(110) == 'CX';
- 	assert cvt.convert(185) == 'CLXXXV';
- 	assert cvt.convert(2002) == 'MMII';
- 	assert cvt.convert(6700) == 'MMMMMMDCC';
+	assert cvt.convert_to_roman(1) == 'I';
+	assert cvt.convert_to_roman(3) == 'III';
+	assert cvt.convert_to_roman(10) == 'X';
+	assert cvt.convert_to_roman(34) == 'XXXIV';
+	assert cvt.convert_to_roman(62) == 'LXII';
+	assert cvt.convert_to_roman(110) == 'CX';
+	assert cvt.convert_to_roman(156) == 'CLVI';
+	assert cvt.convert_to_roman(185) == 'CLXXXV';
+	assert cvt.convert_to_roman(675) == 'DCLXXV';
+	assert cvt.convert_to_roman(2002) == 'MMII';
+	assert cvt.convert_to_roman(3999) == 'MMMCMXCIX';
```

Comment:
You don't test bad input cases like string, non-integer, no input, negative number, over the max range number etc. Also add newline at the end of file.